src/Build/mkdocs.jl

```diff
@@ -0,0 +1,94 @@
+function makemkdocs(build)
+    site_name = "Docile.jl"
+    repo_url = "https://github.com/MichaelHatherly/Docile.jl"
+    site_description = "Julia package documentation system."
+    site_author = "Michael Hatherly"
+    theme = "readthedocs"
+    docs_dir = joinpath(splitdir(Base.source_dir())[end], build)
```

Comment:
What's the plan for allowing these to be user-defined?

One approach might be to leverage multiple dispatch for this kind of thing, ie.

```jl
type MkDocs
    # config fields for mkdocs, "site_name", etc.
    # having a keyword arg constructor for this would be nice
end

function makeconfig(config :: MkDocs, build)
    # ...
end

makeconfig(other, build) = nothing
```

and add a keyword ``config = nothing`` to ``makedocs``. Users then need to provide a ``MkDocs`` object (or any other that we add in future).

So you'd use this like

```jl
makedocs(
    source = "other-dir",
    config = MkDocs(
        site_author = "...",
        theme       = "readthedocs",
    ),
)
```